src/js/components/PluginMountPointComponent.jsx

```diff
@@ -51,7 +60,11 @@ var PluginMountPointComponent = React.createClass({
   },
 
   render: function () {
-    return <div>{this.state.mountPoint}</div>;
+    return (
+      <div className={this.props.className} style={this.props.style}>
```

Comment:
This will produce empty `className` and `style` as these are not required props. Not a big deal, but should we be a bit more defensive?